.gitignore

```diff
@@ -0,0 +1 @@
+!**/env
```

Comment:
Replace this line with `.env`, so the dotenv file is correctly hidden.

Reply:
Ready